shell/browser/native_browser_view_views.cc

```diff
@@ -92,9 +92,35 @@ void NativeBrowserViewViews::ResetAutoResizeProportions() {
 }
 
 void NativeBrowserViewViews::SetBounds(const gfx::Rect& bounds) {
+  // Previous implementation
+  /*
   auto* view = GetInspectableWebContentsView()->GetView();
   view->SetBoundsRect(bounds);
   ResetAutoResizeProportions();
+  */
+
+  /*
+  This implementation is attempting to ignore setting bounds
+  if the view has been destroyed, (ie invalid).
+  A different check is most likely needed as the destroyed
+  pointer cannot be guaranteed NULL.
+  When the pointer is NULL, the program does not crash, but when
+  it is not, it proceeds to still crash.
+  */
+  auto* temp = GetInspectableWebContentsView();
+  LOG(INFO) << "NativeBrowserViewViews::SetBounds - temp: " << temp
+            << " null: " << NULL;
+
+  if (temp != NULL) {
+    LOG(INFO)
+        << "NativeBrowserViewViews::SetBounds - NOT NULL, proceed as usual";
```

Comment:
We probably want to remove logs even though it's a wip :)

In addition - let's name the variable a bit more intuitively/readably, perhaps something like:

```cpp
auto* iwc_view = GetInspectableWebContentsView();
if (!iwc_view)
    return;
auto* view = iwc_view->GetView();
view->SetBoundsRect(bounds);
ResetAutoResizeProportions();
```